qiskit/providers/ibmq/api/clients/experiment.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Client for accessing IBM Quantum Experience experiment services."""
+
+import logging
+from typing import List, Dict, Optional, Union, Tuple
+
+from qiskit.providers.ibmq.credentials import Credentials
+
+from ..rest import Api
+from ..session import RetrySession
+from .base import BaseClient
+
+logger = logging.getLogger(__name__)
+
+
+class ExperimentClient(BaseClient):
+    """Client for accessing IBM Quantum Experience experiment services."""
+
+    def __init__(
+            self,
+            access_token: str,
+            credentials: Credentials
+    ) -> None:
+        """ExperimentClient constructor.
+
+        Args:
+            access_token: IBM Quantum Experience access token.
+            credentials: Account credentials.
+        """
+        self._session = RetrySession(credentials.experiment_url, access_token,
+                                     **credentials.connection_parameters())
+        self.base_api = Api(self._session)
+
+    def experiments(
+            self,
+            backend_name: Optional[str],
+            experiment_type: Optional[str] = None,
+            start_time: Optional[List] = None,
+            device_components: Optional[List[str]] = None,
+            tags: Optional[str] = None
```

Comment:
I can only specify a single tag? Or is the data type `Optional[List[str]]`?

Reply:
My bad, fixed.